docs/install/collecting-logs/README.md

```diff
@@ -62,6 +62,85 @@ tools:
 kubectl exec --namespace logging --stdin --tty --container nginx log-collector-0
 ```
 
+## Local collector
+
+If you are using a local Kubernetes cluster for development (Kind, Docker
+Desktop, or Minikube), you can create a `hostPath` PersistentVolume to store the
+logs on your desktop OS. This will allow you to use all your normal desktop
+tools on the files without needing Kubernetes-specific tools.
+
+The PersistentVolumeClaim will look something like this, but the `hostPath` will
+vary based on your Kubernetes software and host operating system. Some example
+values are documented below.
+
+```yaml
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: shared-logs
+  labels:
+    app: logs-collector
+spec:
+  accessModes:
+    - "ReadWriteOnce"
+  storageClassName: manual
+  claimRef:
+    apiVersion: v1
+    kind: PersistentVolumeClaim
+    name: logs-log-collector-0
+    namespace: logging
+  capacity:
+    storage: 5Gi
+  hostPath:
+    path: /run/desktop/mnt/host/c/Users/evank/logs/
+```
+
+And then you'll need to update the StatefulSet's `volumeClaimTemplates` to
+reference the `shared-logs` volume, like this fragment of yaml:
+
+```yaml
+volumeClaimTemplates:
+  metadata:
+    name: logs
+  spec:
+    accessModes: ["ReadWriteOnce"]
+    volumeName: shared-logs
+```
+
+### Kind
+
+When creating your cluster, you'll need to use a `kind-config.yaml` when
+creating your cluster and specify `extraMounts` for each node, like so:
+
+```yaml
+apiversion: kind.x-k8s.io/v1alpha4
+kind: Cluster
+nodes:
+  - role: control-plane
+    extraMounts:
+      - hostPath: ./logs
+        containerPath: /shared/logs
+  - role: worker
+    extraMounts:
+      - hostPath: ./logs
+        containerPath: /shared/logs
+```
+
+You can then use `/shared/logs` as the `spec.hostPath.path` in your
+PersistentVolume. Note that the directory path `./logs` is relative to the
+directory that the Kind cluster was created in.
+
+### Docker Desktop
+
+Docker desktop automatically creates some shared mounts between the host and the
+guest operating systems, so you only need to know the path to your home
+directory. Here are some examples for different operating systems:
+
+| Host OS | `hostPath` | | Mac OS | `/Users/${USER}` | | Windows |
```

Comment:
Does this table look correct?
In markdown rendered https://github.com/evankanderson/docs/blob/twigs/docs/install/collecting-logs/README.md#docker-desktop
doesn't look correct

Reply:
Yeah, not sure what happened here, there were supposed to be newlines to make that a table.